website/addons/wiki/static/pagedown-ace/Markdown.Editor.js

```diff
@@ -1704,9 +1721,13 @@ var Range = ace.require('ace/range').Range;
 
             buttons.redo = makeButton("wmd-redo-button", getStringAndKey("redo"), "-220px", null);
             buttons.redo.execute = function (manager) { inputBox.session.getUndoManager().redo(); };
+
             makeSpacer(4);
             makeCheckBox("wmd-autocom-toggle", "autocom", "-240px", "Autocomplete");
 
+            makeSpacer(5);
+            buttons.help = makeHelpButton("wmd-help-button", getStringAndKey("help"), "-240px");

```

Comment:
I was suggesting removing the hotkey to open help, since it didn't seem to work. Not sure about the implementation details. I believe @mfraezz is suggesting to use `getString` *because* you removed the hotkey.